### PR TITLE
Add JSON RPC Client

### PR DESCRIPTION
**Pull request by robputt**

### CODE REVIEW

File: hm_pyhelper/miner_json_rpc/client.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from jsonrpcclient import request
+
+
+class Client(object):
+
+    def __init__(self, url='http://localhost:4467'):
+        self.url = url
+
+    def __fetch_data(self, method, **kwargs):
+        response = request(self.url, method, **kwargs)
```
Comment (marvinmarnold) [inline hm_pyhelper/miner_json_rpc/client.py:10]: How are failed requests handled?

File: hm_pyhelper/miner_json_rpc/client.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    def get_region(self):
+        return self.__fetch_data('info_region')
+
+    def get_summary(self):
+        return self.__fetch_data('info_summary')
+
+    def get_peer_addr(self):
+        return self.__fetch_data('peer_addr')
+
+    def get_peer_book(self):
```
Comment (marvinmarnold) [inline hm_pyhelper/miner_json_rpc/client.py:25]: Not clear how to pass the `--all` option

**Review by marvinmarnold — CHANGES_REQUESTED** [review]
I copied the source code to my rockpi and it works! Very cool. Could you add some instructions to the readme that explains the exposed API and how to publish to pypi?

Could you also publish v0.2, I only see v0.1 currently?

**Comment by marvinmarnold** [comment]
Another nit is that I'd prefer if we rebase instead of having merge commits from master.

File: hm_pyhelper/miner_json_rpc/client.py (hunk 2 above)
Comment (robputt) [reply hm_pyhelper/miner_json_rpc/client.py:25]: Returning the entire peer book is crazy, by hard coding "self" we get information about the local miner and it's connections without returning the bloat of disconnected peers in the peer book.

File: hm_pyhelper/miner_json_rpc/client.py (hunk 1 above)
Comment (robputt) [reply hm_pyhelper/miner_json_rpc/client.py:10]: We let the exception bubble up to the person using the library to handle it how they see fit, either they can let it bubble up to 500 their app (in the case of hm-diag) or they can catch it and handle accordingly.

**Review by vpetersson — APPROVED** [review]

**Pull request merged**